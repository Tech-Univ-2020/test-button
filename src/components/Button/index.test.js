import React from 'react';
import {Button} from './index';
import { render } from '@testing-library/react';

describe('the button component', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<Button />);

        expect(asFragment()).toMatchSnapshot();
    });
})