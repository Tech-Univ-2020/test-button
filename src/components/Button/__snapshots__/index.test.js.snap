// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the button component should render correctly 1`] = `
<DocumentFragment>
  <button>
    Bleh
  </button>
</DocumentFragment>
`;
